code/_global_vars/sound.dm

```diff
@@ -108,3 +108,6 @@ GLOBAL_LIST_INIT(glasscrack_sound,\
 		'sound/effects/glass_crack2.ogg',\
 		'sound/effects/glass_crack3.ogg',\
 		'sound/effects/glass_crack4.ogg'))
+
+//This is a global list in interest of optimization; list init is expensive enough for this to be worth doing, even with playsound() not being very hot.
+GLOBAL_LIST_INIT(echo_list, new(18))
```

Comment:
Remove this. `playsound_local()` is not remotely hot enough to require optimizations like this.